### PR TITLE
Create main.yml

### PR DESCRIPTION
**Pull request by damarti**

### CODE REVIEW

**Comment by damarti** [comment]
@ropc would either need access to the settings or need you to put the items inside the yml file into the secrets so that we can test the script

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - uses: actions/checkout@v2
+    
+    - name: Connect to SSH server
+      run: ssh -i ${{secrets.SSH_KEY}} ${{secrets.CONNECTION}}
+
+    - name: Switch to dir and git pull
+      run: cd ${{secrets.REMOTE_DIR}} && git pull
+
+    - name: Run docker-compose to get project running
+      run: sudo docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d --build
```
Comment (ropc) [inline .github/workflows/main.yml:30]: wouldn't this also need to cd into the correct directory?

Comment (damarti) [reply .github/workflows/main.yml:30]: It does on line 27.

Comment (ropc) [reply .github/workflows/main.yml:30]: ah. i thought these run independent of one another

Comment (damarti) [reply .github/workflows/main.yml:30]: they might, lol, i'm actually not sure how this works

**Comment by ropc** [comment]
Since it looks like ssh doesn't accept key input from a string: https://stackoverflow.com/a/12042457

We should do this: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#limits-for-secrets

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -23,15 +23,8 @@ jobs:
     # this decrypts the ssh key and stores it in $HOME/secrets/githubaction-key.pem
     # maybe we should add a step to delete after deploying?
     - name: Decrypt ssh key
-      run: ./.github/scripts/decrypt_secret.sh
-      env:
-        SSH_KEY_PASSPHRASE: ${{ secrets.SSH_KEY_PASSPHRASE }}
```
Comment (ropc) [inline .github/workflows/main.yml]: pretty sure we still have to set this one

Comment (damarti) [reply .github/workflows/main.yml]: done

**Pull request merged**